### ISSUE TITLE
An ADA issue: «7. Insufficient color contrast»

### ISSUE THREAD

**Issue by dmitry-fedyuk** [kitchencollection-com/core#3]
Several instances of insufficient color contrast were found across different pages of the Kitchen
Collection website such as ‘My Cart’ link, price information of product, footer headings.

[upwork.com/ab/f/contracts/20805325](https://www.upwork.com/ab/f/contracts/20805325)
[https://www.kitchencollection.com](https://www.kitchencollection.com)

![01](https://user-images.githubusercontent.com/83204/46410299-3077fa80-c73a-11e8-9a03-8e92cdfb4b63.png)
![02](https://user-images.githubusercontent.com/83204/46410359-703ee200-c73a-11e8-96aa-2188275ed766.png)
![03](https://user-images.githubusercontent.com/83204/46410385-8056c180-c73a-11e8-8a19-7446bdf7686a.png)

### Code snippet:
```
<a class="action showcart"…</a>
```
```
<span class="price">$8.97</span>
```
```
<div class="row">
<div class="col-sm-6">
<div class="footer-box">…</div>
```

### Rationale
Two colors are considered to have good visibility if the contrast ratio between foreground text and
background is sufficient. W3C has described the minimum contrast ratio for text that is less than
18 point (if not bold) and less than 14 point (if bold) to be 4.5:1 at Level AA and 7:1 at Level AAA.

### Recommendations
On all the pages of the website, ensure that each text/background color combination presents a
contrast ratio of at least 4.5:1 for standard text, and 3.0:1 for large text (18 points or 14 points and
bold). 
The [Color Contrast Analyzer](http://www.paciellogroup.com/resources/contrast-analyser.html) tool allows the user to determine whether a combination of foreground and background colors meets the minimum color contrast ratio.

**Comment by dmitry-fedyuk** [comment]
I have decomposed it to separate tasks:

- [An ADA issue: the «MY CART» title in the top-right corner has an insufficient color contrast](https://github.com/kitchencollection-com/core/issues/6)

**Comment by dmitry-fedyuk** [comment]
Fixed in [0.0.6](https://github.com/kitchencollection-com/core/releases/tag/0.0.6).
